src/assessment.js

```diff
@@ -49,5 +49,12 @@ var q2 = [/* your answer here */];
 // executing it.
 //
 var q5 = function fireOnAlderaan () {
-  // your code here
+  return deathStar.call(fire);
```

Comment:
As written, this will not work, because `deathStar` does not have a `call` method - only functions do. You are correct, however, in thinking that `call` will be part of the answer.